### PR TITLE
[MO] - [system test] -> MirrorMaker, UserInDifCluster

### PR DESCRIPTION
**Pull request by Seequick1**
### Type of change

- Enhancement / new feature
- Refactoring

### Description

This PR adding 2 STs. First one user that is created in kafka cluster1 will be unable to do changes in another specific cluster. Second one is to test whitelist in MirrorMaker. Also some basic refactor added.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [x] Write tests
- [x] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java [hunk 1]
```diff
@@ -78,6 +77,24 @@ void testDeployUndeploy() {
         verifyLabelsForServiceAccounts(KAFKA_CLUSTER_NAME, null);
     }
 
+    void testDockerImagesForKafkaConnect() {
+        LOGGER.info("Verifying docker image names");
+        //Verifying docker image for cluster-operator
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java]: is it a comment that should be removed?

Comment (Seequick1) [reply systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java]: Yeah. I will remote it :).

File: systemtest/src/main/java/io/strimzi/systemtest/MessagingBaseST.java [hunk 2]
```diff
@@ -79,6 +79,45 @@ void availabilityTest(int messageCount, long timeout, String clusterName, boolea
...
+    /**
+     * Method for send messages to specific kafka cluster. It uses test-client API for communication with deployed clients inside kubernetes cluster
+     * @param messageCount messages count
+     * @param timeout timeout for producer to be finished
+     * @param clusterName cluster name
+     * @param tlsListener option for tls listener inside kafka cluster
+     * @param topicName topic name
+     * @return count of send and acknowledged messages
+     */
+    int sendMessages(int messageCount, long timeout, String clusterName, boolean tlsListener, String topicName) throws Exception {
```
Comment (scholzj) [inline systemtest/src/main/java/io/strimzi/systemtest/MessagingBaseST.java]: Out fo curiosity . don't we have some methods for sending and receiving messages elsewhere? It seems to me like we have to use this already somewhere in other tests.

File: systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java [hunk 3]
```diff
@@ -78,6 +77,22 @@ void testDeployUndeploy() {
         verifyLabelsForServiceAccounts(KAFKA_CLUSTER_NAME, null);
     }
 
+    void testDockerImagesForKafkaConnect() {
```
Comment (scholzj) [inline systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java]: Why are we moving this / changing from private? What changed?

**Comment by scholzj** [comment]
@strimzi-ci run tests testcase=ConnectST,MirrorMakerST,KafkaST

**Comment by Seequick1** [comment]
@strimzi-ci run tests testcase=ConnectST,MirrorMakerST,KafkaST

File: systemtest/src/main/java/io/strimzi/systemtest/MessagingBaseST.java (hunk 2 above)
Comment (Seequick1) [reply systemtest/src/main/java/io/strimzi/systemtest/MessagingBaseST.java]: Correct, it was just overloaded method without using KafkaUser parameter.

**Comment by Seequick1** [comment]
@strimzi-ci run tests testcase=ConnectST,MirrorMakerST,KafkaST

**Pull request merged**